torch_em/data/datasets/medical/motum.py

```diff
@@ -0,0 +1,141 @@
+"""The MOTUM dataset contains annotations for tumor (brain metastases and high grade glioma) segmentation
+in brain multi-modal MRI scans.
+
+The dataset is located at https://doi.gin.g-node.org/10.12751/g-node.tvzqc5/.
+This dataset is from the publication https://doi.org/10.1038/s41597-024-03634-0.
+Please cite it if you use this dataset for your research.
+"""
+
+import os
+from glob import glob
+from natsort import natsorted
+from typing import Union, Tuple, Literal, List
+
+from torch.utils.data import Dataset, DataLoader
+
+import torch_em
+
+from .. import util
+
+
+URL = "https://doi.gin.g-node.org/10.12751/g-node.tvzqc5/10.12751_g-node.tvzqc5.zip"
+CHECKSUM = ""
```

Comment:
What about the checksum here? Does setting it to `""` actually work? I would assume that it fails because it has to be `None` if no checksum is provided?!

Reply:
I forgot to add that. I've updated it in the latest commit now. Thanks for spotting!